src/bidiMapper/domains/log/logHelper.spec.ts

```diff
@@ -19,7 +19,7 @@ import {expect} from 'chai';
 
 import type {Script} from '../../../protocol/protocol.js';
 
-import {logMessageFormatter} from './logHelper.js';
+import {getRemoteValuesText, logMessageFormatter} from './logHelper.js';
```

Comment:
Merged both Test files as they are not that big + its more readable to have one src <-> one spec. Happy to revert if that is not the case.

Reply:
LGTM